app/models/ext_management_system.rb

```diff
@@ -482,15 +484,29 @@ def vm_count_by_state(state)
     vms.inject(0) { |t, vm| vm.power_state == state ? t + 1 : t }
   end
 
-  def total_vms_on;        vm_count_by_state("on");        end
+  def host_count_by_metric(metric)
+    total_metric = hosts.inject(0) { |t, host| t + host.send(metric) }
```

Comment:
It will be great, I wanted make in the same way of the others. If @kbrock is agree with that I'll make the change, on the other hand What do you think about have a header with information about total_cores, total_memory and total_instances/total_hosts in that view?
And make this changes too in cloud views?